docs/content/configuration/csi_ug_config_create_vol_snapshotclass.md

```diff
@@ -17,18 +16,24 @@ When configuring the file, be sure to use the same array secret and array secret
     
 -   The `pool` parameter is not available on IBM FlashSystem A9000 and A9000R storage systems. For these storage systems the snapshot must be created on the same pool as the source.
 
-```screen
+```
 apiVersion: snapshot.storage.k8s.io/v1beta1
 kind: VolumeSnapshotClass
 metadata:
-  name: demo-snapshotclass
+  name: demo-volumesnapshotclass-config-secret
 driver: block.csi.ibm.com
 deletionPolicy: Delete
 parameters:
-  csi.storage.k8s.io/snapshotter-secret-name: demo-secret
+  # non-csi.storage.k8s.io parameters may be specified in by_system_id per system and/or outside by_system_id as the cross-system default.
+
+  by_system_id: '{"demo-system-id-1":{"pool":"demo-pool-1","SpaceEfficiency":"deduplicated","snapshot_name_prefix":"demo-prefix-1"},
+                  "demo-system-id-2":{"pool":"demo-pool-2","snapshot_name_prefix":"demo-prefix-2"}}'  # Optional.
+  pool: demo-pool                    # Optional. Use to create the snapshot on a different pool than the source.
+  SpaceEfficiency: thin              # Optional. Use to create the snapshot with a different space efficiency than the source.
+  snapshot_name_prefix: demo-prefix  # Optional.
+
+  csi.storage.k8s.io/snapshotter-secret-name: demo-config-secret
```

Comment:
@ArbelNathan do we really want to remove the non-config usage from our docs?

Reply:
@ArbelNathan gentle reminder

Reply:
it should be in addition, not to replace the old..

Reply:
it looks like a replacement to me.
should we have a separate configuration topic for topology (similar to `Advanced configuration`)?

Reply:
we can, with all the topology specific object

Reply:
@oriyarde  @ArbelNathan  Following up offline for now

Reply:
DONE